api/client/samples/crop_models/brazil_soybeans.ipynb

```diff
@@ -42,10 +42,11 @@
      "name": "stdout",
```

Comment:
<p>shouldn't the dataframe be filtered so that there is only a single yield value for each year, to avoid double counting in the regression?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
If I understand this correctly, I think it's intended to be a weighted regression so we can split the difference on points where we have disagreement between the sources. But I don't see anything in here providing the weights - I think we'd have to give 1/n where n is number of available points for that date. Unless it's intended to double-count to simulate added confidence when multiple sources agree. Not 100% sure the intent, so I'll leave this for @nemozen to comment on

Reply:
right the intention is to use *all* the series for yield when there's more than one, this is an improvement over taking any one series. we are taking advantage of the fact that we have different sources and giving them equal weight.